aptos-move/vm-genesis/src/lib.rs

```diff
@@ -350,7 +350,16 @@ fn publish_stdlib(session: &mut SessionExt<impl MoveResolver>, stdlib: Modules)
 }
 
 /// Trigger a reconfiguration. This emits an event that will be passed along to the storage layer.
-fn reconfigure(session: &mut SessionExt<impl MoveResolver>) {
+fn emit_new_block_and_epoch_event(session: &mut SessionExt<impl MoveResolver>) {
```

Comment:
nit: Should we name this reconfigure_and_emit_genesis_block_event to make it clear it's also calling reconfigure rather than just emitting an event (which implies no changes potentially made on chain)?

Reply:
I think it only emits the event, it calls emit_genesis_reconfiguration_event